setup.py

```diff
@@ -33,6 +33,6 @@
         'Programming Language :: Python :: 3.7',
     ],
     packages=find_packages(),
-    python_requires='>=2.7',
```

Comment:
Let's keep `python_requires` unchanged at the moment.

Reply:
Currently (in this PR) this library isn't compatible with 3.8, wouldn't you want to have that explicitly stated so tools like pipenv/pip-tools/conda can use this information to negotiate compatibility and dependency resolution? If your plan is to deploy a new version to PyPI incorporating changes from both this and #14 then disregard this message.

Reply:
Yes, I'm going to also incorporate other changes, then I'll release a new version.